app/frontend/action_plans/action_plan_show.component.js

```diff
@@ -116,8 +96,35 @@ class ActionPlanShow extends Component {
                 namespace="action_plans"
                 useServerLinks={ true }/>
 
-              <ActionPlanReviewer />
+              <ActionPlanReviewer visible={this.state.editable} />
+
+              <h2>Estat de l'actuacio</h2>
+              <p>TODO</p>
```

Comment:
lol XD, that's not even finished! We need to talk about the other PR @josepjaume